ddmrp_report_part_flow_index/views/ddmrp_flow_index_group_views.xml

```diff
@@ -19,20 +19,6 @@
         <field name="arch" type="xml">
             <form>
                 <sheet>
-                    <div class="oe_button_box" name="button_box">
-                        <button
-                            name="toggle_active"
-                            type="object"
-                            class="oe_stat_button"
-                            icon="fa-check"
-                        >
-                            <field
-                                name="active"
-                                widget="boolean_button"
-                                options='{"terminology": "active"}'
-                            />
-                        </button>
```

Comment:
Typically it is replaced by:

```
<widget
                        name="web_ribbon"
                        title="Archived"
                        bg_color="bg-danger"
                        attrs="{'invisible': [('active', '=', True)]}"
                    />
```

(not exaclty in the same position, you can check usages in other form views)

Reply:
Thanks for the comments, I have applied the changes!